### PR TITLE
Get Account info feature

### PR DESCRIPTION
**Pull request by HarryL5004**
Issue #173

### CODE REVIEW

File: internal/producer/src/accounts/accounts.go [hunk 1]
```diff
@@ -392,3 +340,98 @@ func ValidateContract(c *Contract, table string, authorizedMinters [][]byte) (bo
...
+	serializedAccount := make([]byte, 16) // 8 + 8 bytes for balance and stateNonce
+	binary.LittleEndian.PutUint64(serializedAccount[:8], accInfo.balance)
+	binary.LittleEndian.PutUint64(serializedAccount[8:], accInfo.stateNonce)
+	return serializedAccount, nil
+}
+
+func (accInfo *AccountInfo) Deserialize(serializedAccountInfo []byte) error {
+	// accInfo = &AccountInfo{
+	// 	balance:    binary.LittleEndian.Uint64(serializedAccountInfo[:8]),
+	// 	stateNonce: binary.LittleEndian.Uint64(serializedAccountInfo[8:])}
```
Comment (kstola2) [inline internal/producer/src/accounts/accounts.go]: What is the purpose of these comments?

**Comment by HarryL5004** [comment]
Oh, I thought you meant removing duplicate codes in `account` when you said `replace any code in the accounts package that this feature overlaps with` in the issue. That's why I made the changes to the previous functions...

File: internal/producer/src/accounts/accounts.go (hunk 1 above)
Comment (HarryL5004) [reply internal/producer/src/accounts/accounts.go]: Oh, I forgot to remove these comments before committing.. Sorry about that

**Comment by kstola2** [comment]
@HarryL5004 Yeah that's correct what you did I was just double checking the intent behind it. Great work. Just remove those comments in the convo and I'll approve.

**Pull request merged**